doc/user_guide/user_scripts.rst

```diff
@@ -102,9 +104,11 @@ is applied). These are:
     # List of all files that psyclone will skip processing
     FILES_TO_SKIP = ["broken_file1.f90", "broken_file2.f90"]
 
-    # Boolean to decide whether PSyclone should chase external modules while
-    # creating a PSyIR tree in order to obtain better external symbol information.
-    # It can also be a list of module names for more precise control
+    # Whether to chase the imported modules to improve symbol information (it can
+    # also be a list of module filenames chase only specific modules). This has to
```

Comment:
"...filenames to limit the chasing to only specific modules"

Reply:
Done